Neos.Fusion/Resources/Private/Fusion/Root.fusion

```diff
@@ -167,6 +167,65 @@ prototype(Neos.Fusion:ResourceUri) {
   @exceptionHandler = 'Neos\\Fusion\\Core\\ExceptionHandlers\\AbsorbingHandler'
 }
 
+# Renders a link pointing to a controller/action
+#
+# Usage:
+# link = Neos.Fusion:Link.Action {
+#     package = 'Some.Package'
+#     controller = 'Standard'
+#     action = 'index'
+# }
+#
+prototype(Neos.Fusion:Link.Action) < prototype(Neos.Fusion:Component) {
```

Comment:
I'm a total noob with Fusion but I hope this works as I expect. The expectation is, that the Fusion:Link works like a normal anchor tag, with only some extended attributes that are passed to the inner href UriBuilder.

So
`<Neos.Fusion:Link.Action class="foo" target="_blank" action="hello" controller="world" ...>Hello World!</Neos.Fusion:Link.Action>`

Reply:
This doesn't work for me, it creates a tag without any attributes: https://fusionpen.punkt.de/fusionpen/4ae7f6eb68d5a7cf0b0c36ff050a4d40dd2835b4.html

Probably because it's missing the `attributes.` before `href` below.